phoenix_liveview/assets/app.js

```diff
@@ -0,0 +1,7 @@
+import "./app.css"
+import { Socket } from "phoenix"
```

Comment:
The imports from hex packages has been a source of pain (see `NODE_PATH` hack further below), but it works.